src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -1065,3 +1065,31 @@ It can use the following patterns: +
 
  * *maxDeploymentFailure:* number of attempts when starting a ProActive Node on the cluster using the nsSubmitCommand, after all attempts failed, the ProActive Node will be declared as `Lost`.
 
+ * *singleConnectionUserName:* user name for single SSH connection. Single SSH connection credentials can be used when setting impersonation method to _sudo_ or _none_. It must be used also when enabling the native scheduler metrics acquisition (see below).
+
+ * *singleConnectionPassword:* password for single SSH connection. Single SSH connection credentials can be used when setting impersonation method to _sudo_ or _none_. It must be used also when enabling the native scheduler metrics acquisition (see below).
```

Comment:
What is impersonation ?

Reply:
this term is explained in the corresponding parameter documentation